### DISCUSSION TITLE
install

### DISCUSSION THREAD

**Discussion by hossam-create** [karrioapi/karrio#362] [General]
how to install ?and how to make it in my e commerce website

**Comment by danh91** [comment]
Hi @hossam-create, 
Can you tell us more about your site? 

- Are you using an e-commerce platform? (saleor, woocommerce, medusa or Shopify?)
- Are you hosting it yourself?
- Does it have an API?

**Comment by hossam-create** [comment]
hi,
my web site is classified ads and auction
host on shared hosting

   - Does it have an API?
   - i don't know

mnbarh.com

**Reply by danh91** [reply]
I see. How do you process your shipment right now?

**Comment by hossam-create** [comment]
I need to make your software in my sits so with every purchase from buyers
the sell can do it and administrator can track shipping
